apps/demo.namerank.io/app/results.tsx

```diff
@@ -0,0 +1,85 @@
+import { createClient } from "@namehash/namerank";
+import { Indicator } from "./indicator";
+
+const namerank = createClient({
+  namerankEndpoint:
+    "https://izzkysqb6d6qzhnpv4ybqyty2e0ktjwe.lambda-url.us-east-1.on.aws/namerank",
```

Comment:
I created a separate story for this: https://app.shortcut.com/ps-web3/story/25981/fix-endpoint-for-namerank